R/survivalFunctions.R

```diff
@@ -7,7 +7,8 @@
 #' @examples
 #' ExpSurvPFS(c(1:5), 0.2, 0.4)
 ExpSurvPFS <- function(t, h01, h02) {
-  assert_numeric(t, lower = 0)
+  assert_vector(t, all.missing = FALSE)
+  assert_true(all(t >= 0))
```

Comment:
I don't understand yet I think why this is needed, because `checkmate::assert_numeric(c(0, 1, 2), lower = 0)` works fine?